geniza/corpus/sitemaps.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
+
+from geniza.corpus.models import Document
+from geniza.corpus.solr_queryset import DocumentSolrQuerySet
+
+
+def solr_timestamp_to_date(timestamp):
+    """Convert solr isoformat date time string to python date."""
+    # format: 2020-05-12T15:46:20.341Z
+    # django sitemap only includes date, so strip off time
+    yearmonthday = timestamp.split("T")[0]
+    return date(*[int(val) for val in yearmonthday.split("-")])
+
+
+class DocumentSitemap(Sitemap):
```

Comment:
Did you try setting `i18n = True` and `alternates = True` ? It would be nice if we didn't have to change anything with our document sitemaps when we're ready to enable other languages.

Reply:
Looks like alternates isn’t implemented until Django 3.2? When I set it to true, I got a strange plain text file locally. [The sitemap framework | Django documentation | Django](https://docs.djangoproject.com/en/4.0/ref/contrib/sitemaps/#django.contrib.sitemaps.Sitemap.alternates)

I turned on `i18n`. As I explained elsewhere, I was getting thousands of results until I turned that on (?!) so after setting `i18n = True` I added the `.get_results(rows=100000)`

Reply:
We're on django 3.2; I don't know why the sitemap renders as text instead of xml when we turn on the alternates behavior, the content type still looks correct. I guess we should leave it off for now since it doesn't seem to work correctly.